<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Member</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true">QHeaderView::section {
    background-color: #ebebeb;
}</string>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row/>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Summary</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Due Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Estimate</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time Spent</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_taskEdit">
      <property name="title">
       <string>Log Time</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_id">
         <property name="text">
          <string>    Task ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_id">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_summary">
         <property name="text">
          <string>    Summary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_summary">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_time">
         <property name="text">
          <string>    Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_time_day">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_time_day">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_time_hour">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_time_hour">
         <property name="text">
          <string>hours</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QPushButton" name="button_submit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
